cutobjects/PbPb/CutsDstoKKpi_PbPb2018_3050_Raa.C

```diff
@@ -956,6 +964,7 @@ AliRDHFCutsDstoKKpi* MakeFileForCutsDs3050_Central2018_Pass3(bool fUseStrongPID
     esdTrackCuts->SetRequireTPCRefit(true);
     esdTrackCuts->SetRequireITSRefit(true);
     esdTrackCuts->SetMaxChi2PerClusterTPC(2.5);
+    esdTrackCuts->SetMinNCrossedRowsTPC(70);

```

Comment:
Isn't this cut applied in the filtering?

Reply:
It should be in the FB4, but I could not find the list of cuts in the FB4, so to be 100% sure I added it (it does not hurt in any case)

Reply:
It is in FB4 https://twiki.cern.ch/twiki/pub/ALICE/PWGPPAODTrackCuts/rs_aodbits_130214_copy.pdf, https://github.com/alisw/AliRoot/blob/master/ANALYSIS/ANALYSISalice/AliESDtrackCuts.cxx#L821

Reply:
Ciao Fabio, thanks a lot for checking! I have remove it and I reverted the mtehods for the Ds in pass1 for bookkeeping as you suggested (I did not do it for the D+, which we mainly use for checks)